.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
@johntkeller we do not want to include the ".vs" folder. I think we should add that to the .gitignore.